### DISCUSSION TITLE
I can't open the admin page

### DISCUSSION THREAD

**Discussion by krasho** [markshust/docker-magento#788] [General]
Hello.
I have been using your docker environment and suddenly I couldn't open the admin page. it seems Magento is trying to find the admin page as a regular page.

Versions
- Magento 2.4.0
- markoshust/magento-nginx:1.18-5
- markoshust/magento-php:7.3-fpm
- mariadb:10.4
- markoshust/magento-elasticsearch:7.9.3-1
- redis:6.0-alpine

I tried to create a new environment in a new folder and installed the project on a different computer; the problem was the same.

I followed some recommendations that I saw on different pages.
- Delete folders
- Check the name to the backend
- fix permissions
- check the htaccess file but I think this is to apache, not Nginx
- I changed the database and I tried open the page with https://domain/index.php/admin
- I checked the logs and I couldn't find errors there

please help me

regards

**Comment by rangerz** [comment]
viewing log is in `bin/bash` to `ls var/log/`

Also check your admin URL: `bin/magento info:adminuri`

**Reply by krasho** [reply]
hello rangerz

I checked both:
- there aren't errors in var/log
- I am getting "admin" value when I execute **bin/magento info:adminuri** 

do you have more recommendations?

thanks
